graal/backends/core/analyzers/scc.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*- the Graal backend.
+#
+# Copyright (C) 2015-2019 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     Nishchith Shetty <inishchith@gmail.com>
+#
+
+import subprocess
+
+from graal.graal import GraalRepository
+from .analyzer import Analyzer
+
+
+class SCC(Analyzer):
+    """A wrapper for SCC.
+
+    This class allows to call SCC over a file, parses
+    the result of the analysis and returns it as a dict.
+    """
+    version = '0.1.0'
+
+    def __init__(self):
+        pass
+
+    def __analyze_file(self, message):
+        """Add information about LOC, blank and commented lines using SCC for a given file
+
+        :param message: message from standard output after execution of SCC
+
+        :returns result: dict of the results of the analysis over a file
+        """
+
```

Comment:
Please remove this empty line. If the docstring is one line, there should be a blank line between the docstring and the code, otherwise there shouldn't be any blank line